src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -170,6 +170,9 @@ protected String downloadSignatureScanner(File scannerExpansionDirectory, HttpUr
         BlackDuckRequestBuilder requestBuilder = new BlackDuckRequestBuilder()
                 .url(downloadUrl)
                 .addHeader("Version", localScannerVersion);
+        if (!localScannerVersion.isEmpty()) {
+            requestBuilder.addHeader("Accept-Version", "ANY");
+        }
```

Comment:
Fixes IDETECT-4455. With the addition of this header, the /api/tools (new) scan-cli download request will behave as the old API did where upgrades/downgrades from any installed version to any other version was allowed. (Accept-Version: ANY header can only be provided IF we are sending a non-empty version header, otherwise we get a 404.)